scripts/setup.sh

```
@@ -22,6 +34,128 @@ metadata:
+
+# Optionally install LVMS as storage service (must be before keycloak which needs a default storage class)
+if [[ "${STORAGE_SERVICE}" == "true" ]]; then
+    wait_for_namespace_cleanup openshift-storage
+    echo "Installing LVMS storage service..."
+    oc apply -f prerequisites/lvms/lvms-operator.yaml
+    retry_until 300 3 '[[ -n "$(oc get csv --no-headers -n openshift-storage | grep lvms)" ]]' 'oc apply -f prerequisites/lvms/lvms-operator.yaml || true' || {
+        echo "Timed out waiting for LVMS CSV to exist"
+        exit 1
+    }
+    LVMS_CSV=$(oc get csv --no-headers -n openshift-storage | awk '/lvms/ { print $1 }' | tail -1)
+    wait_for_resource clusterserviceversion/${LVMS_CSV} jsonpath='{.status.phase}'=Succeeded 300 openshift-storage
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)